src/System.Linq.Parallel/tests/QueryOperators/AnyTests.cs

```diff
@@ -0,0 +1,150 @@
+        public static void Any_AllFalse(Labelled<ParallelQuery<int>> labelled, int count)
+        {
+            ParallelQuery<int> query = labelled.Item;
+            IntegerRangeSet seen = new IntegerRangeSet(0, count);
+            Assert.False(query.Any(x => !seen.Add(x)));
+            Assert.True(seen.Complete());
+        }
+
+        [Theory]
+        [OuterLoop]
+        [MemberData("Ranges", (object)(new int[] { 1024 * 1024, 1024 * 1024 * 4 }), MemberType = typeof(UnorderedSources))]
+        public static void Any_AllFalse_Longrunning(Labelled<ParallelQuery<int>> labelled, int count)
+        {
+            Any_AllFalse(labelled, count);
+        }
+
+        [Theory]
+        [MemberData("Ranges", (object)(new int[] { 0, 1, 2, 16 }), MemberType = typeof(UnorderedSources))]
+        public static void Any_AllTrue(Labelled<ParallelQuery<int>> labelled, int count)
+        {
+            ParallelQuery<int> query = labelled.Item;
+            Assert.Equal(count > 0, query.Any(x => x >= 0));
+        }
+
+        [Theory]
+        [OuterLoop]
+        [MemberData("Ranges", (object)(new int[] { 1024 * 1024, 1024 * 1024 * 4 }), MemberType = typeof(UnorderedSources))]
+        public static void Any_AllTrue_Longrunning(Labelled<ParallelQuery<int>> labelled, int count)
+        {
+            Any_AllTrue(labelled, count);
+        }
+
+        [Theory]
+        [MemberData("OnlyOneData", (object)(new int[] { 2, 16 }))]
+        public static void Any_OneFalse(Labelled<ParallelQuery<int>> labelled, int count, int position)
+        {
+            ParallelQuery<int> query = labelled.Item;
+            Assert.True(query.Any(x => !(x == position)));
+        }
+
+        [Theory]
+        [OuterLoop]
+        [MemberData("OnlyOneData", (object)(new int[] { 1024 * 1024, 1024 * 1024 * 4 }))]
+        public static void Any_OneFalse_Longrunning(Labelled<ParallelQuery<int>> labelled, int count, int position)
+        {
+            Any_OneFalse(labelled, count, position);
+        }
+
+        [Theory]
+        [MemberData("OnlyOneData", (object)(new int[] { 2, 16 }))]
+        public static void Any_OneTrue(Labelled<ParallelQuery<int>> labelled, int count, int position)
+        {
+            ParallelQuery<int> query = labelled.Item;
+            Assert.True(query.Any(x => x == position));
+        }
+
+        [Theory]
+        [OuterLoop]
+        [MemberData("OnlyOneData", (object)(new int[] { 1024 * 1024, 1024 * 1024 * 4 }))]
+        public static void Any_OneTrue_Longrunning(Labelled<ParallelQuery<int>> labelled, int count, int position)
+        {
+            Any_OneTrue(labelled, count, position);
+        }
+
+        //
+        // Tests the Any() operator applied to infinite enumerables
+        //
+        [Fact]
+        public static void Any_Infinite()
+        {
+            Assert.True(InfiniteEnumerable().AsParallel().Any());
+            Assert.True(InfiniteEnumerable().AsParallel().Any(x => true));
+        }
```

Comment:
You missed it.  In fact, I test (almost?) everything with an empty source (sometimes this means they get `Assert.Throws...` instead)